### ISSUE TITLE
Fix bikeshed link errors for implementation-defined and url

### ISSUE THREAD

**Issue by dlrobertson** [whatwg/fetch#1580]
## Summary

There appear to be some link errors that bikeshed throws some warnings for the following, due to multiple definitions:

 - [x] `url` definitions ([concept-url from url](https://url.spec.whatwg.org/#concept-url) and [navigation-status-url from webdriver](https://w3c.github.io/webdriver-bidi/#navigation-status-url))
 - [ ] `implementation defined` definitions ([implementation-defined from infra](https://infra.spec.whatwg.org/#implementation-defined) and [implementation-defined from ecmascript](https://tc39.es/ecma262/#implementation-defined))

## Example

```
 make local
bikeshed spec fetch.bs fetch.html --md-Text-Macro="COMMIT-SHA LOCAL COPY"
LINK ERROR: Multiple possible 'url' dfn refs for '['/']'.
Arbitrarily chose https://url.spec.whatwg.org/#concept-url
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:url; type:dfn; text:url
spec:webdriver-bidi; type:dfn; text:url
<a data-link-for="/" data-link-type="dfn" data-lt="URL">URL</a>
LINK ERROR: Multiple possible 'implementation-defined' dfn refs.
Arbitrarily chose https://infra.spec.whatwg.org/#implementation-defined
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:infra; type:dfn; text:implementation-defined
spec:ecmascript; type:dfn; text:implementation-defined
<a data-link-type="dfn" data-lt="implementation-defined">implementation-defined</a>
LINK ERROR: Multiple possible 'urls' dfn refs for '['/']'.
Arbitrarily chose https://url.spec.whatwg.org/#concept-url
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:url; type:dfn; text:url
spec:webdriver-bidi; type:dfn; text:url
<a data-link-for="/" data-link-type="dfn" data-lt="URLs">URLs</a>
 ✔  Successfully generated, with 3 linking errors
```

**Comment by whimboo** [comment]
The underlying issue with the WebDriver BiDi spec should be fixed now (https://github.com/w3c/webdriver-bidi/issues/347)

**Comment by dlrobertson** [comment]
> The underlying issue with the WebDriver BiDi spec should be fixed now ([w3c/webdriver-bidi#347](https://github.com/w3c/webdriver-bidi/issues/347))

Thanks!
